client/src/components/games/AvailableGames.tsx

```diff
@@ -94,22 +96,32 @@ function AvailableGames() {
           <div className='filterDate'>
             <label htmlFor='date'>DATE & TIME</label>
             <br />
-            <input type='datetime-local' name='date' id='filterDate' value={dateFilter} onChange={handleDateFilterChange} />
+            <input
+              type='datetime-local'
+              name='date'
+              id='filterDate'
+              value={dateFilter}
+              onChange={handleDateFilterChange}
+            />
           </div>
 
         </form>
       </div>
       <div className='availableGamesContainer'>
 
         {filteredGames.length > 0 && filteredGames.map(game => (
-          <div className='availableGame' key={game._id}>
-            <NavLink to={`/game-details/${game._id}`}>
+          <div className='availableGame' key={game.id}>

```

Comment:
Related to my question in game.d.ts
Do you have a property of id here? Because the API call will return _id.